### ISSUE TITLE
Preview Window not updating 

### ISSUE THREAD

**Issue by shawnwildermuth** [RickStrahl/MarkdownMonster#463]
In 1.13.12, my preview window is no longer updating as I type. I've tried the different sync types and none of them seem to update without manually turning preview on and off. Known issue or can I give you some logging to help?

**Comment by shawnwildermuth** [comment]
Only updates once I change the sync status, but never continues to update. Seems to be only on Weblog posts. I'm going to try and replicate the behavior.

**Comment by shawnwildermuth** [comment]
So it seems to be specific to both this document and my rendering theme. I can't find anything specific in the document that is causing it though. Very confusing.

**Comment by shawnwildermuth** [comment]
It seems specific to a document I had. I'm going to close it as it's too hard to replicate, probably not a general problem and I got past the issue.

**Comment by RickStrahl** [comment]
Ok. Any hint on what it was what triggered it? It would be good to find out  to at least be aware  what's causing problems like this.

**Comment by shawnwildermuth** [comment]
It worked fine with your theme, but not my theme (which I built from my website theme). So it has more to do with the code I have running in the theme than your engine. Still not sure which it is but I'm happy to send you the theme and post if you want to play with it.

**Comment by RickStrahl** [comment]
Sure that would be fine - be worth taking a look. No guarantees that I can make it work but I'd like to see what exactly fails. My guess is that there's a script error happening and that stops the page from completing its proper load which in turn may prevent the preview from updating properly.

**Comment by shawnwildermuth** [comment]
Here are the files:

[ShouldYouBeUsingVueCli.zip](https://github.com/RickStrahl/MarkdownMonster/files/2670932/ShouldYouBeUsingVueCli.zip)
[PreviewThemes.WilderMinds.zip](https://github.com/RickStrahl/MarkdownMonster/files/2670947/PreviewThemes.WilderMinds.zip)

**Comment by RickStrahl** [comment]
Took a look.

You removed the script code that's responsible for updating the preview, so that pretty much explains why it's not working 😄. The script code dynamically updates the browser and handles preview syncing and interop with the page. Without that, the initial rendering works, but that's it - refreshes or at least scrolled refreshes aren't going to work.

All themes automatically load Bootstrap so you shouldn't need that from your local folder. All you should have to do to an existing theme is replace the `theme.css` with your's if you made changes and that should just work. If you have script that needs to run you can add it to the bottom of the page.

Going to take a look at your previous examples.

**Comment by RickStrahl** [comment]
Adding the `preview.js` script and highlightjs code back in the preview works as expected even with your previous themes.

Looks crappy as it's missing margins, but it works :-)

**Comment by shawnwildermuth** [comment]
I'll take a look. The problem with BS is that I'm on BS4, not 3. And I am not using the Highlight that you're using. Let me try again with preview.js.

**Comment by shawnwildermuth** [comment]
Yes, it was not only the preview.js but the id on the script. But that was the fix. Thanks!
